src/include/duckdb/function/register_function_list_helper.hpp

```diff
@@ -30,6 +30,36 @@ static void FillFunctionParameters(FunctionDescription &function_description, co
 	}
 }
 
+static vector<string> GetExamplesForFunctionAlias(const string &function_name, const string &alias_of,
+                                                  vector<string> &all_examples) {
+	vector<string> filtered_examples;
+	bool is_operator = (!function_name.empty() && !(function_name[0] >= 'a' && function_name[0] <= 'z') &&
+	                    !(function_name[0] >= 'A' && function_name[0] <= 'Z'));
+	bool alias_of_is_operator = (!alias_of.empty() && !(alias_of[0] >= 'a' && alias_of[0] <= 'z') &&
+	                             !(alias_of[0] >= 'A' && alias_of[0] <= 'Z'));
```

Comment:
There is some code to check for operators in Tokenizer.hpp, e.g. `BaseTokenizer::isSpecialOperator` and `BaseTokenizer::IsSingleByteOperator`. Maybe they could be made static/reusable here in a future PR?